lib/simplifyExpression/basicsSearch/basicRules.js

```diff
@@ -0,0 +1,224 @@
+const {build, query} = require('math-nodes');
+const {rewriteNode} = require('math-rules');
+
+const ChangeTypes = require('../../ChangeTypes');
+const checks = require('../../checks');
+const defineRuleString = require('../../util/defineRuleString');
+const Node = require('../../node');
+
+// e.g. -(-3) -> 3
+const NEGATION = defineRuleString('--#a', '#a');
```

Comment:
I wonder if this should deal with chains all at once 

e.g. should -----5 go to ---5 or -5 ?